Mathlib/LinearAlgebra/Finsupp/Defs.lean

```diff
@@ -127,6 +127,10 @@ theorem lapply_comp_lsingle_same (a : α) : lapply a ∘ₗ lsingle a = (.id : M
 theorem lapply_comp_lsingle_of_ne (a a' : α) (h : a ≠ a') :
     lapply a ∘ₗ lsingle a' = (0 : M →ₗ[R] M) := by ext; simp [h.symm]
 
+include R in
+lemma apply_surjective (a : α) : Function.Surjective fun f : α →₀ M ↦ f a :=
+  LinearMap.surjective_of_comp_eq_id (R := R) _ _ <| lapply_comp_lsingle_same a
```

Comment:
This doesn't need `R`, see https://github.com/leanprover-community/mathlib4/pull/23997#discussion_r2042055010